blog/en/blog/2023/11/09/api-versioning.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Resizing images on-the-fly
```

Comment:
Why this title

Reply:
Because I copy-pasted and forgot to update. Good catch, thanks!